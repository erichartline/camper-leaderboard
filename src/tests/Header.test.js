import React from 'react';
import ReactDOM from 'react-dom';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../components/Header';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
});
