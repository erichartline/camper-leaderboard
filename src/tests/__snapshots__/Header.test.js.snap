// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="header"
>
  <h2>
    freeCodeCamp 
    <i
      aria-hidden="true"
      className="fa fa-free-code-camp fcc-icon"
    />
     
    <br />
    Camper Leaderboard
  </h2>
</div>
`;
